tests/test_models.py

```diff
@@ -395,6 +395,35 @@ def test_mars_local_tms():
     assert bbox == syrtis_tms.bbox
 
 
+def test_mars_tms_construction():
+    mars_sphere_crs = pyproj.CRS.from_user_input("IAU_2015:49900")
+    extent = [-90, -180, 90, 180]
+    mars_tms = morecantile.TileMatrixSet.custom(
+        extent, mars_sphere_crs, id="MarsGeographicCRS"
```

Comment:
```suggestion
        extent, 
        mars_sphere_crs, 
        id="MarsGeographicCRS", 
        matrix_scale=[1, 2],
```

it's `[1, 2]` and not `[2, 1]` because the axis is inverted